### ISSUE TITLE
Add a layer which overlays the OSM GPS data onto the map

### ISSUE THREAD

**Issue by busykit** [openstreetmap/openstreetmap-website#1239]
could be implemented as a checkbox in the layers menu.

Rationale: There are Areas where quite some GPS tracks are available which are not yet mapped. See for instance:

http://mc.bbbike.org/mc/?lon=119.918553&lat=28.459032&zoom=12&num=1&mt0=mapnik&mt-1=ol_osm-gps&mt-1p=70&marker=

I know that a similar feature is already included in the iD editor, but it is well hidden there. Making the GPS traces available on the main web page would possibly encourage mapping these traces or at least help to get orientation in not so well mapped areas.

**Comment by tomhughes** [comment]
I'm not sure this is practical given the volume of data involved - it would be very expensive to be constantly querying the database for the points as the map was moved and trying to draw thousands of points would probably break many browsers.

**Comment by busykit** [comment]
Well, of course you would not want to render each GPS point on it's own.

My idea was to have pre rendered partially transparent PNG layers which will be re-rendered on a regular basis. let's say each week or each day.

Just like it is done in the link that I gave.

**Comment by tomhughes** [comment]
Well that's a massive project, and out of scope for this repository. Were such tiles to exist we could use them but providing them is not in scope here really.

**Comment by pnorman** [comment]
Aren't we running GPS tiles for iD already?

**Comment by tomhughes** [comment]
Well there is http://gps.tile.openstreetmap.org/ yes but that's not everything as I think it only includes those ones where the raw trace is public? Is that what iD uses?

**Comment by bhousel** [comment]
iD uses [this from editor-layer-index](https://github.com/osmlab/editor-layer-index/blob/c7e57a63c6988fae7567af7f581675eb5c06757c/sources/world/OpenStreetMap-GPS.json)

` "url": "http://{switch:a,b,c}.gps-tile.openstreetmap.org/lines/{zoom}/{x}/{y}.png",`

**Comment by SomeoneElseOSM** [comment]
Yes - public traces only, and there are other problems with that layer - missing traces and "dashed traces":

http://b.gps-tile.openstreetmap.org/lines/15/16260/10651.png

(both issues already logged in what is hopefully the right place)
